exist-core/src/test/java/org/exist/xquery/functions/util/Base64FunctionsTest.java

```diff
@@ -66,5 +66,28 @@ public void testBase64EncodeDecode() throws XMLDBException {
         assertEquals("This is a test!", r);
     }
 
+    @Test
+    public void testBase64EncodeUrlSafeNoSpecial() throws XMLDBException {
```

Comment:
should we have a test showing that the old 2 param function still works as it did previously?

Reply:
Well, I will add it. I thought the two parameter function had no tests and is deprecated now what is the use. But I see why you would want that.

Reply:
test added